test/e2e/common_suite_test.go

```diff
@@ -246,7 +255,14 @@ func (tc *testCase) run() {
 							break
 						}
 					}
+				}
+
+				if tc.isAuth {
+					if err := getAuthenticatedImageStatus(ctx, client, tc.AuthImageStatus, *tc.pod); err != nil {
+						t.Fatal(err)
+					}
 
+					t.Logf("PodVM has successfully reached %v state with authenticated Image - %v", tc.AuthImageStatus, os.Getenv("AUTHENTICATED_REGISTRY_IMAGE"))
```

Comment:
Is there are reason that isAuth section was moved from it's original place below the `if tc.expectedPodLogString` check?

Reply:
No, this might happened while rebasing and resolving conflicts, and it got added back

Reply:
Can you move it to the original place please to keep the diff simpler. Thanks